core/src/main/resources/hudson/Messages.properties

```diff
@@ -110,6 +110,7 @@ ProxyConfiguration.TestUrlRequired=Test URL is required.
 ProxyConfiguration.MalformedTestUrl=Malformed Test URL {0}.
 ProxyConfiguration.FailedToConnectViaProxy=Failed to connect to {0}.
 ProxyConfiguration.FailedToConnect=Failed to connect to {0} (code {1}).
+ProxyConfiguration.NonTLSWarning=Please note that this value may be exposed because of the non-TLS communication with the proxy
```

Comment:
💡 Is "be exposed" sufficient in terms of information? I mean, as someone configuring it, do I understand easily "where"/"to who" it is exposed?

Reply:
I tried to keep it short, but I can try to make it more clear. How about this one?

```
Please note that the connection between Jenkins and the proxy is always via an HTTP connection. Therefore, this insecure connection may expose the credentials provided.
```

Reply:
As it's a warning, the conciseness is not "that" important I would say. The proposal works better for me :)

Reply:
@Wadeck I updated the messages: e0cf56459ca6c8d0d732358e4de27955d5d7858e